packages/Chem/src/widgets/properties.ts

```diff
@@ -56,6 +85,70 @@ export function getMoleculeCharge(mol: OCL.Molecule): number {
   return moleculeCharge;
 }
 
+export async function statsWidget(molCol: DG.Column<string>): Promise<DG.Widget> {
+  const host = ui.div();
+  host.style.marginLeft = '-25px';
+  const size = Math.min(molCol.length, 1000);
+  const randomIndexes = Array.from({length: size}, () => Math.floor(Math.random() * molCol.length));
+
+  function getAverage(p: IChemProperty) {
+    const propertiesArray = new Array(size);
+    for (let idx = 0; idx < randomIndexes.length; ++idx) 
+      propertiesArray[idx] = getPropertyValue(molCol, randomIndexes[idx], p);
+    const col = DG.Column.fromList(p.type, molCol.dataFrame.columns.getUnusedName(p.name), propertiesArray);
+
+    const addColumnIcon = getIcon(p, molCol); 
+
+    ui.tools.setHoverVisibility(host, [addColumnIcon]);
+    $(addColumnIcon).addClass('chem-plus-icon');
+    addColumnIcon.style.top = '10px';
+
+    const bins = DG.BitSet.create(col.length);
+    bins.setAll(true);
+    const hist = histogram(col, bins, true, {bins: 20, logScale: false});
+  
+    const histRoot = ui.div();
+    histRoot.classList.add('chem-distribution-hist');
+  
+    const hw = 100;
+    const hh = 35;
+    const m = 4;
+
+    const canvas = ui.canvas(hw, hh);
+    const ctx = canvas.getContext('2d', {willReadFrequently : true});
+    
+    if (ctx) 
+      renderMultipleHistograms(ctx, new DG.Rect(m, m, hw - 2 * m, hh - 2 * m), [hist], {fill: false, markerSize: 1});
+    histRoot.appendChild(canvas);
+    
+    const colAvg = StringUtils.formatNumber(col.stats.avg);
+    const colStdev = StringUtils.formatNumber(col.stats.stdev);
+    return ui.divH([
+      addColumnIcon, 
+      ui.divText(`${colAvg} ± ${colStdev}`, {style: {'width': '100px', 'position': 'relative', 'top': '8px'}}),
+      histRoot
+    ], { style: {'position': 'relative'}});
```

Comment:
same here

Reply:
same as the comment before, don't see reason for putting single styles to a separate class